app.rb

```diff
@@ -37,12 +47,12 @@ def email_insight(phone_info, email)
           :via_options => {
             :address              => "smtp.gmail.com",
             :port                 => 587,
-            :user_name            => 'my@email.com',
-            :password             => MY_PASSWORD,
+            :user_name            => ENV['GMAIL_USERNAME'],
```

Comment:
Can we assign the env variable values to constants and use those constants in the application? It'll reduce the lookup for the value to just the single execution.

Reply:
Absolutely!  Updated to use Sinatra's settings.  Also refactored out the global `$vonage` variable to a simple caching method.

Reply:
Awesome, looks great!